src/shogun/optimization/FirstOrderSAGCostFunction.h

```diff
@@ -31,8 +31,8 @@
 
```

Comment:
As you are already touching this file, it would be nice to do a tiny extra effort and update the Copyright to the more modern shorter version.